src/cordova/plugin/OutlineAndroidLib/outline/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id 'com.android.library'
+}
+
+android {
+    namespace 'org.outline'
+    compileSdk 30
+
+    defaultConfig {
+        minSdk 22
+        targetSdk 30
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles "consumer-rules.pro"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled true
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+        debug {
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
+
+dependencies {
+
+    // To make this project editable directly in the source tree, we need to load
+    // tun2socks.aar from its original path in the third_party directory.  We do
+    // this by relative path:
+    compileOnly fileTree(dir: '../../../../../third_party/outline-go-tun2socks/android', include: ['tun2socks.aar'])
```

Comment:
Perhaps we add a symlink to the aar file in the OutlineAndroidLib directory so we don't need this hack?
I think Cordova will copy the content of the symlink.